wmn-data.json

```diff
@@ -2135,6 +2189,16 @@
         "known" : ["johnny", "dev"],
         "cat" : "coding"
        },
+       {
+        "name" : "GOG",
+        "uri_check" : "https://www.gog.com/u/{account}",
+        "e_code" : 200,
+        "e_string" : "window.profilesData.profileUser",
+        "m_code" : 404,

```

Comment:
I'm getting a 302 code. Please confirm.

Reply:
I created https://github.com/WebBreacher/WhatsMyName/issues/900 because the initial request receives a 302 status code, which then redirects to a page that returns a 404 error. Python's libraries follow the redirect, leading to the final 404 page.

Reply:
While Python libraries do, the vast majority of the users that use WMN do so via a web browser which will read a 301/302 and not the 404 page for the immediate result. In the past, WMN has treated 301/302 redirects as the result and not the pages that they redirect to. Please adjust this site for that.

Reply:
It's ready but then let's expand the scheme